paths/electron/distributing/03-sharing-on-websites.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: simple-class
+header:
+  overlay_image: cover.jpeg
+  overlay_filter: rgba(46, 129, 200, 0.6)
+title: Sharing on Websites
+permalink: /electron/distributing-apps/sharing-on-websites
+next-page: /electron/distributing-apps/celebrate
+facilitator: false
+sidebar:
+  nav: "electron-sharing-apps"
+main-content: |
+
+  ## Preparing Electron Apps for Download
+
+
+  Most commonly, electron apps are shared via [GitHub Releases](https://help.github.com/articles/about-releases/). People who create and package their own applications upload the binaries to a GitHub repository where others can download them. You can see some great examples of this on the [electron app page](https://electron.atom.io/apps/). This is free, version controlled, and kept cleanly alongside your code.
+
+  You could then link to your downloads from your very own [GitHub pages site](https://pages.github.com/), or any site for that matter. [Learn how to make your own GitHub Pages site here](https://services.github.com/on-demand/github-cli/).
+
+  We recommend [this help documentation for creating releases on GitHub](https://help.github.com/articles/creating-releases/).
```

Comment:
Here - we should specify exactly which files people should upload. When I did it on Mac, I just compressed the entire directory that was created and uploaded the `.zip`, but I don't think that was accurate. ❓commenting here to remember to go back and look at existing electron apps release pages to see what they've uploaded.